Lib/fontmake/font_project.py

```diff
@@ -541,6 +540,12 @@ def save_otfs(
                 if "GSUB" in gsub_src:
                     font["GSUB"] = gsub_src["GSUB"]
 
+            # Read autohinting parameters from ufo lib if present
+            if "com.schriftgestaltung.customParameter.InstanceDescriptorAsGSInstance.TTFAutohint options" in ufo.lib and autohint is None:
+                autohint = ufo.lib["com.schriftgestaltung.customParameter.InstanceDescriptorAsGSInstance.TTFAutohint options"]
```

Comment:
I would put this into a separate variable, because here you are pulling out a per-master setting and using it to override the settings for the whole project. If the next master in the loop doesn't have autohint options,  `autohint` will contain the settings from the previous master.